lookout/style/format/feature_extractor.py

```diff
@@ -235,6 +235,74 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None) \
                  the corresponding `VirtualNode`-s and the parents mapping \
                  or None in case no features were extracted.
         """
+        parsed_files, node_parents, vnode_parents = self._parse_vnodes(files, lines)
+        xy = self._files_to_xy(parsed_files)
+        if xy is None:
+            return None
```

Comment:
actually, I do not like we return None in this case. We either can return empty lists and dicts or raise an exception and catch it in a caller.

But I think you do not want to change feature extractor interface in this PR. So we can keep it for now. 

please tell me if you think it is ok as it is.

Reply:
Extracting zero vnodes is definitely not an error, so yep, we should return empty lists. I leave it for you :)